src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/HtmlConfiguration.java

```diff
@@ -115,6 +128,23 @@
 
     private final HtmlOptions options;
 
+    /**
+     * Kinds of pages.
+     */
+    // Note: this should (eventually) be merged with Navigation.PageMode,
+    // which performs a somewhat similar role
+    public enum PageKind {
```

Comment:
That would be a reasonable short term fix, but I was hoping to merge with `Navigation.PageMode` since both are enums for the different kinds of pages we generate. But I have not yet found the sweet spot for where to put this enum: it may be best for it to end up as a top-level enum in the `formats.html` package.  Or, we avoid merging them, and rename `PageKind` as you suggest.